index.js

```diff
@@ -20,3 +21,5 @@ module.exports = function (messageFile) {
     process.exit(1)
   }
 }
+
+module.exports()
```

Comment:
This seems pointless - let's remove the function and put the code inside on the top level, running it immediately.